jax/interpreters/pxla.py

```diff
@@ -299,7 +249,9 @@ def indices(self, shape: Tuple[int, ...]) -> np.ndarray:
               .transpose(perm))
 
   def __eq__(self, other):
-    return (self.sharding, self.mesh_mapping) == (other.sharding, other.mesh_mapping)
+    return (self.sharding, self.mesh_mapping) == (other.sharding,
+                                                  other.mesh_mapping)
+
 
```

Comment:
nit: lots of whitespace between methods

Reply:
I guess it depends on the conventions there:
- I am working from google3, the required spacing is 2 new lines between functions (1 between methods).
- In Jax, it seems more frequent to have one line between functions, but it depends.

I have been conforming to google3 style when submitting code, and in practice, I adapt to each reviewer (but I do have a small preference for google3 style, given my day to day work, deals with these conventions), but I can understand too if JAX wants to enforce a different style -- in which case, I would encourage the team to configure some linter, that will say what it should be, and does not let each individual decides what they prefer.

Given that, what do you prefer?

Reply:
This was a method, removed one new line (the next CL is turning it into a function).